packages/slate-history/test/undo/cursor/keep_after_focus_and_remove_text_undo.js

```diff
@@ -0,0 +1,49 @@
+/** @jsx jsx */
+
+import assert from 'assert'
+import { Transforms, Editor } from 'slate'
+import { jsx } from '../..'
+
+export const run = editor => {
+  // focus at the end
+  Transforms.select(editor, {
+    anchor: { path: [0, 0], offset: 5 },
+    focus: { path: [0, 0], offset: 5 },
+  })
+  // select all
+  Transforms.select(editor, {
+    anchor: { path: [0, 0], offset: 5 },
+    focus: { path: [0, 0], offset: 0 },
+  })
+  // remove
+  Editor.deleteFragment(editor)
+  // blur
+  Transforms.deselect(editor)
+  // focus back
+  Transforms.select(editor, {
+    anchor: { path: [0, 0], offset: 0 },
+    focus: { path: [0, 0], offset: 0 },
+  })
+}
+
+export const input = (
+  <editor>
+    <block>Hello</block>
+  </editor>
+)
+
+export const output = {
```

Comment:
I've tried to use hyperscript for this, but couldn't make it work because the expected `output.children` have contained an extra empty text node for:
```
export const output = (
  <editor>
    <block>Hello</block>
    <cursor />
  </editor>
)
```
and I'm not sure how to fix this.